src/bitcoind.js

```diff
@@ -51,10 +51,10 @@ export function bitcoindParams(client) {
 export async function bitcoindListUnspent({url, auth, address, addresses}) {
   try {
     const addressParam = addresses || [address]
-    const resp = await callBitcoind(url, auth, 'listunspent', [0, 9999999, addressParam], );
+    const resp = await callBitcoind(url, auth, 'listunspent', [0, 9999999, addressParam]);
     const promises = [];
     resp.result.forEach(utxo => {
-      promises.push(callBitcoind(url, auth, 'getrawtransaction', [utxo.txid, 1]))
```

Comment:
`getrawtransaction` is used for mempool txs or if a block is specified. We are assuming that the nodes we are querying are probably not using any kind of indexing, but will import addresses they want to track with caravan. for that `gettransaction` should be used (i.e. for wallet txs)